content/content-team-interview-with-jan-david-nose-rust-infrastructure-team/index.md

```diff
@@ -0,0 +1,32 @@
++++
+path = "2025/11/22/content-team-interview-with-jan-david-nose-rust-infrastructure-team"
+title = "Content Team: Interview with Jan David Nose, Rust Infrastructure Team"
+authors = ["Pete LeVasseur"]
+
+[extra]
+team = "Content Team"
+team_url = "https://rust-lang.org/governance/teams/launching-pad/#team-content"
++++
+
+We had our first whirlwind outing as the [Rust Project Content Team] at RustConf 2025 in Seattle, Washington, USA. There we had a chance to speak with folks about interesting things happening in the Rust Project and wider Rust Community.
+
+# Jan David Nose, Rust Infrastructure Team
+
+In this interview, [Xander Cesari] sits down with [Jan David Nose], one of the full-time engineers on the [Rust Infrastructure Team]. JD and the team maintain and develop the infrastructure upon which Rust is developed and deployed, including a whole host of CI/CD tooling for developing the language itself as well as the infra behind crates.io that serves the community and users. We're releasing this video on an accelerated timeline in light of the recent software supply chain attacks, but the interview was conducted prior to the recent news of compromised packages in other languages and ecosystems.
+
+Check out the [interview here] or click below.
+
+<p style="text-align: center;">
+  <a href="https://youtu.be/r7i-2wHtNjw"
+     target="_blank" rel="noopener noreferrer">
+    <img src="jan-david-nose-interview.png"
+         alt="Interview with Jan David Nose – Rust Infrastructure Team"
+         style="max-width: 100%; height: auto;">
+  </a>
+</p>
```

Comment:
At 650KB, the PNG file seems a bit large for what it's doing for us here, both in terms of what we need to add to the Git repo and what the user needs to load for the page.  Also, the UX is a bit non-obvious here, as it almost looks like an embed but there's no play button.

Might it be better to just use a YT embed link here?  Alternatively, the plain link would probably be OK.

Reply:
Thanks.  I squashed that onto the commit where the PNG was originally added so the Git repo doesn't need to carry the deleted file.